hyperspy/misc/hyperspy_magics.py

```diff
@@ -26,10 +37,9 @@ def hyperspy(self, line):
 
         Positional arguments:
         ---------------------
-            toolkit
+            toolkit : {qt4, gtk, wx, tk, None}
```

Comment:
No, we should not add the quotation marks - everything after the magic is passed as a string, and if user actually writes `"qt4"` it will fail, as it's supposed to be.

Also, the only recognised flat is `-r` - if anything else is passed, it just stops at toolkit check (where we check if the passed toolkit is supported).

Finally, can we really say that we allow gtk and tk? Or just "it won't crash, but won't work either"? If they don't give full functionality, I see no point allowing them